### ISSUE TITLE
completion: CTRL+Space is not applicable in Delete/Backspace triggered session

### ISSUE THREAD

**Issue by ivanbasov** [dotnet/roslyn#35301]
Both old and new completions. 
Start with 

```C#
class C
{
    void Method()
    {
        var s = "";
        s.Length$$
    }
}
```

Intellisense TriggerOnDelete should be ON
SendBackspace
SendCommitIfUnique

**Expected**
No active completion session
`s.Length` is in the current line

**Actual**
Active completion session
`s.Lengt` is in the current line

**Comment by ivanbasov** [comment]
This is a follow up for https://github.com/dotnet/roslyn/pull/35310. It seems there is no need to fix it in 16.1. The issue is not a regression and happened long ago.

**Comment by CyrusNajmabadi** [comment]
Closing out as speculative.
